pkgs/applications/misc/prusa-slicer/default.nix

```
@@ -188,6 +186,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   cmakeFlags = [
     "-DSLIC3R_STATIC=0"
+    "-DDSLIC3R_PCH=0"
```

```suggestion
    "-DSLIC3R_PCH=0"
```
Though maybe it isn't needed after all, if it builds as is